src/report.ts

```diff
@@ -15,25 +15,32 @@ export const logReport = (
 	}
```

Comment:
this function is very nested try to use the return early pattern

Reply:
You have to help me here!
I don't want to return, I want to execute depending on things...

Code style... is always tricky.
Let me know how to improve

Reply:
i can do it